adoc/retail_chap_formulas.adoc

```diff
@@ -0,0 +1,209 @@
+[[retail.chap.formulas]]
+= Formulas
+
+include::entities.adoc[]
+
+
+Formulas are pre-written Salt states, that are used to configure your {smr} installation.
+
+This section lists the primary formulas shipped with {smr} and their configuration options.
+
+
+.State and formula name collisions
+[IMPORTANT]
+====
+If a formula uses the same name as an existing Salt state, the two names will collide, and could result in the formula being used instead of the state.
+Always check the names of states and formulas to avoid name collisions.
+====
+
+Most formulas can be updated using the {susemgr} {webui}.
+Once you have made changes to your formula, ensure you apply the highstate to propagate your changes to the appropriate services.
+
+
+// We should ensure these are in alphabetical order. LKB
+
+[[retail.sect.formulas.branch-network]]
+== Branch Network Formula
```

Comment:
As this formula description is now here, it should be removed from [administration chapter](https://github.com/SUSE/doc-susemanager/blob/develop/adoc/retail_chap_admin.adoc#branch-server-network-configuration).